src/main/steps/enter-your-address/content.ts

```diff
@@ -46,8 +46,41 @@ const en = () => {
   };
 };
 
-// @TODO translations
-const cy = en;
+const cy = () => {
+  const addressPostcode = {
+    required: 'Nid ydych wedi rhoi cod post. Rhowch god post cyn parhau.',
+    invalid: 'Nid ydych wedi rhoi cod post yn y DU dilys. Rhowch god post yn y DU dilys cyn parhau.',
+    notSelected: 'Nid ydych wedi rhoi eich cyfeiriad. Rhowch eich cyfeiriad cyn parhau.',
```

Comment:
These 3 lines look the same in enter-your-address and enter-their-address so could maybe go in common content

Reply:
They're slightly different. `enter-your-address` is "Your postcode" and `enter-their-address` is "Your ${partner}'s postcode", so there wouldn't be any reuse between them here